kokoro/scripts/linux/build-docker.sh

```diff
@@ -142,7 +142,7 @@ elif [ $TOOL = "cmake-smoketest" ]; then
   cmake -GNinja -DRE2_BUILD_TESTING=OFF -DCMAKE_BUILD_TYPE="Release" ..
 
   echo $(date): Build glslang...
-  ninja glslangValidator
+  ninja glslang
```

Comment:
Is the Cmake target is actually glslang-standalone? 

See https://github.com/KhronosGroup/glslang/pull/3257/files

Reply:
Indeed you have to invoke "glslang-standalone" to build the executables

```
dneto@dneto1:/build/shaderc/build$ ls -l third_party/glslang/StandAlone/
total 248
drwxr-x--- 4 dneto dneto   4096 Jul 19 18:24 CMakeFiles
-rw-r----- 1 dneto dneto   3669 Jul 19 18:24 cmake_install.cmake
-rw-r----- 1 dneto dneto    347 Jul 19 18:24 CTestTestfile.cmake
-rw-r----- 1 dneto dneto    374 Jul 19 18:24 glslang-standaloneTargets.cmake
-rwxr-x--- 1 dneto dneto 232144 Jul 19 18:26 spirv-remap
-rw-r----- 1 dneto dneto    370 Jul 19 18:24 spirv-remapTargets.cmake
dneto@dneto1:/build/shaderc/build$ ninja glslang
ninja: no work to do.
dneto@dneto1:/build/shaderc/build$ ls -l third_party/glslang/StandAlone/
total 248
drwxr-x--- 4 dneto dneto   4096 Jul 19 18:24 CMakeFiles
-rw-r----- 1 dneto dneto   3669 Jul 19 18:24 cmake_install.cmake
-rw-r----- 1 dneto dneto    347 Jul 19 18:24 CTestTestfile.cmake
-rw-r----- 1 dneto dneto    374 Jul 19 18:24 glslang-standaloneTargets.cmake
-rwxr-x--- 1 dneto dneto 232144 Jul 19 18:26 spirv-remap
-rw-r----- 1 dneto dneto    370 Jul 19 18:24 spirv-remapTargets.cmake
dneto@dneto1:/build/shaderc/build$ ninja glslang-standalone
[1/1] Linking CXX executable third_party/glslang/StandAlone/glslang
dneto@dneto1:/build/shaderc/build$ ls -l third_party/glslang/StandAlone/
total 8804
drwxr-x--- 4 dneto dneto    4096 Jul 19 18:24 CMakeFiles
-rw-r----- 1 dneto dneto    3669 Jul 19 18:24 cmake_install.cmake
-rw-r----- 1 dneto dneto     347 Jul 19 18:24 CTestTestfile.cmake
-rwxr-x--- 1 dneto dneto 8760696 Jul 19 18:31 glslang
-rw-r----- 1 dneto dneto     374 Jul 19 18:24 glslang-standaloneTargets.cmake
lrwxrwxrwx 1 dneto dneto       7 Jul 19 18:31 glslangValidator -> glslang
-rwxr-x--- 1 dneto dneto  232144 Jul 19 18:26 spirv-remap
-rw-r----- 1 dneto dneto     370 Jul 19 18:24 spirv-remapTargets.cmake
```